pkgs/tools/system/bottom/default.nix

```
@@ -17,7 +17,7 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  buildInputs = lib.optional stdenv.hostPlatform.isDarwin darwin.apple_sdk.frameworks.IOKit;
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [darwin.apple_sdk.frameworks.IOKit libiconv];
```

```suggestion
  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ IOKit libiconv ];
```
Darwin shouldn't be used as an input.